snapcraft/plugins/nodejs.py

```diff
@@ -78,19 +78,22 @@ def schema(cls):
 
         # Inform Snapcraft of the properties associated with building. If these
         # change in the YAML Snapcraft will consider the build step dirty.
-        schema['build-properties'].extend(['node-packages', 'node-engine'])
+        schema['build-properties'].append('node-packages')
+        # Inform Snapcraft of the properties associated with pulling. If these
+        # change in the YAML Snapcraft will consider the build step dirty.
```

Comment:
I don't understand this comment twice. Kyle is doing the same, is it only me who sees it unnecessary?